src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java

```diff
@@ -52,18 +52,36 @@ private void process() {
 
 		switch (dataProcessId) {
 
-		case 2:
-			System.out.println("How many records do you want to insert ? : ");
-			int empCount = scanner.nextInt();
+		case 1:
+			System.out.println("###### " + DriverTypes.EMPLOYEE.getName().toUpperCase() + " Processor Starts ######");
+
+			Arrays.stream(OperationTypes.values()).forEach(eachOperation -> {
+				System.out.println(eachOperation.getOperationType() + " : " + eachOperation.getOperationName());
+			});
+			System.out.println("Please choose the operation type: ");
+			int operationTypeEmployee = scanner.nextInt();
+
+			if (operationTypeEmployee == 1) {
+
+				System.out.println("How Many Records do you want to insert ? :");
+				int employeeCount = scanner.nextInt();
 
-			int empRowCount = 1;
-			EmployeeDataProcessor empProcessor = new EmployeeDataProcessor(scanner, empDao);
-			for (int i = 0; i < empCount; i++) {
-				empProcessor.process(empRowCount);
-				empRowCount++;
+				EmployeeDataProcessor emp = new EmployeeDataProcessor(scanner, empDao);
+				int empCount = 1;
+				for (int i = 0; i < employeeCount; i++) {
+					emp.process(empCount);
+					empCount++;
+				}
+				System.out.println(employeeCount +"records inserted sucessfully");
+			} else if (operationTypeEmployee == 2) {
+				System.out.println("fetching employee records from the database");
+				EmployeeInsertDao empInsertDao=new EmployeeInsertDao();
```

Comment:
dont create object here , use the instance block initialized empDao reference to call the method.

Reply:
yes sir i removed